### DISCUSSION TITLE
Next/Prev Space and Pycharm and VSCode

### DISCUSSION THREAD

**Discussion by petervines** [rxhanson/RectanglePro-Community#462] [Q&A]
Next/Prev space works with all my apps except Pycharm and VSCode. When I try to move a Pycharm or VSCode window to say the next space. The desktop switches to the next space however the Pycharm or VSCode window stays where it is instead of moving as well. Is there some way to make this work? I have seen similar issues raised in other discussions, but none specifically for Pycharm or VSCode.

**Comment by rxhanson** [comment]
Thanks for reporting. There is delicate timing involved with this, and I'll revisit it for the next release to see what I can do.

**Reply by pcwa-ahendricks** [reply]
Not sure if this is helpful but I'm experiencing the same issue with VSCode. Thanks for adding this feature by the way, it's very helpful and I'm certain macOS didn't make it easy for you (devs).

**Comment by rxhanson** [comment]
I've made some adjustments in v3.0.20 that should help with this. In my testing, it was working fine with VSCode, but Jet Brains IDEs were still a little finicky.
